flow-report/App.tsx

```diff
@@ -5,38 +5,40 @@
  */
 
 import {FunctionComponent} from 'preact';
-import {useState} from 'preact/hooks';
+import {Sidebar} from './Sidebar';
+import {FlowResultContext, useCurrentLhr} from './util';
 
 export const Report: FunctionComponent<{lhr: LH.Result}> = ({lhr}) => {
   // TODO(FR-COMPAT): Render an actual report here.
   return (
-    <div>
+    <div data-testid="Report">
       <h1>{lhr.finalUrl}</h1>
       {
         Object.values(lhr.categories).map((category) =>
-          <h2>{category.id}: {category.score}</h2>
+          <h2 key={category.id}>{category.id}: {category.score}</h2>
         )
       }
     </div>
   );
 };
 
+export const Summary: FunctionComponent = () => {
+  // TODO(FR-COMPAT): Design summary page.
+  return <h1 data-testid="Summary">SUMMARY</h1>;
+};
+
+const Content: FunctionComponent = () => {
```

Comment:
We may have a content `<div>` in the future, also needed to use the `useCurrentLhr()`